articles/iot-hub/iot-hub-devguide-security.md

```diff
@@ -292,7 +292,7 @@ You can use any X.509 certificate to authenticate a device with IoT Hub. Certifi
 
 * **An existing X.509 certificate**. A device may already have an X.509 certificate associated with it. The device can use this certificate to authenticate with IoT Hub.
 * **A self-generated and self-signed X-509 certificate**. A device manufacturer or in-house deployer can generate these certificates and store the corresponding private key (and certificate) on the device. You can use tools such as [OpenSSL][lnk-openssl] and [Windows SelfSignedCertificate][lnk-selfsigned] utility for this purpose.
-* **CA-signed X.509 certificate**. You can also use an X.509 certificate generated and signed by a Certification Authority (CA) to identify a device and authenticate a device with IoT Hub.
+* **CA-signed X.509 certificate**. You can also use an X.509 certificate generated and signed by a Certification Authority (CA) to identify a device and authenticate a device with IoT Hub. IoTHub will only verify that the thumbprint presented matches the configured thumbprint. IotHub does not validate the certificate chain.
```

Comment:
Can you re-word this slightly to conform to our style guide:
"IoT Hub only verifies that the thumbprint presented matches the configured thumbprint. IoT Hub does not validate the certificate chain."

Thanks!